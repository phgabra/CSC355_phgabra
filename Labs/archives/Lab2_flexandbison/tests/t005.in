int main() {
    int a = 5;
    int b = 10, c = 20;

    float e = 3.14;
    float f = 5.67E8;
    float x = 4.56e-3;

    char h = 'H';
    char i = 'I';

    char j[] = "Hello, World!";
    char *k = "This is a test.";

    int l = 1, m = 2, n = 3;
    float o = 4.56, p = 7.89;

    int *q = &a;
    float *r = &e;

    int s;
    float t;
    char u;
    int *w;

    return 0;
}

