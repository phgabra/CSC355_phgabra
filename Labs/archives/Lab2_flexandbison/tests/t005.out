<T_KEYWORD, 'int'>
<T_IDENTIFIER, 'main'>
<T_DELIMITER, '('>
<T_DELIMITER, ')'>
<T_DELIMITER, '{'>
<T_KEYWORD, 'int'>
<T_IDENTIFIER, 'a'>
<T_OPERATOR, '='>
<T_NUMBER, '5'>
<T_DELIMITER, ';'>
<T_KEYWORD, 'int'>
<T_IDENTIFIER, 'b'>
<T_OPERATOR, '='>
<T_NUMBER, '10'>
<T_DELIMITER, ','>
<T_IDENTIFIER, 'c'>
<T_OPERATOR, '='>
<T_NUMBER, '20'>
<T_DELIMITER, ';'>
<T_KEYWORD, 'float'>
<T_IDENTIFIER, 'e'>
<T_OPERATOR, '='>
<T_NUMBER, '3.14'>
<T_DELIMITER, ';'>
<T_KEYWORD, 'float'>
<T_IDENTIFIER, 'f'>
<T_OPERATOR, '='>
<T_NUMBER, '5.67E8'>
<T_DELIMITER, ';'>
<T_KEYWORD, 'float'>
<T_IDENTIFIER, 'x'>
<T_OPERATOR, '='>
<T_NUMBER, '4.56e-3'>
<T_DELIMITER, ';'>
<T_KEYWORD, 'char'>
<T_IDENTIFIER, 'h'>
<T_OPERATOR, '='>
<T_CHAR_LIT, ''H''>
<T_DELIMITER, ';'>
<T_KEYWORD, 'char'>
<T_IDENTIFIER, 'i'>
<T_OPERATOR, '='>
<T_CHAR_LIT, ''I''>
<T_DELIMITER, ';'>
<T_KEYWORD, 'char'>
<T_IDENTIFIER, 'j'>
<T_DELIMITER, '['>
<T_DELIMITER, ']'>
<T_OPERATOR, '='>
<T_STRING_LIT, '"Hello, World!"'>
<T_DELIMITER, ';'>
<T_KEYWORD, 'char'>
<T_OPERATOR, '*'>
<T_IDENTIFIER, 'k'>
<T_OPERATOR, '='>
<T_STRING_LIT, '"This is a test."'>
<T_DELIMITER, ';'>
<T_KEYWORD, 'int'>
<T_IDENTIFIER, 'l'>
<T_OPERATOR, '='>
<T_NUMBER, '1'>
<T_DELIMITER, ','>
<T_IDENTIFIER, 'm'>
<T_OPERATOR, '='>
<T_NUMBER, '2'>
<T_DELIMITER, ','>
<T_IDENTIFIER, 'n'>
<T_OPERATOR, '='>
<T_NUMBER, '3'>
<T_DELIMITER, ';'>
<T_KEYWORD, 'float'>
<T_IDENTIFIER, 'o'>
<T_OPERATOR, '='>
<T_NUMBER, '4.56'>
<T_DELIMITER, ','>
<T_IDENTIFIER, 'p'>
<T_OPERATOR, '='>
<T_NUMBER, '7.89'>
<T_DELIMITER, ';'>
<T_KEYWORD, 'int'>
<T_OPERATOR, '*'>
<T_IDENTIFIER, 'q'>
<T_OPERATOR, '='>
<T_OPERATOR, '&'>
<T_IDENTIFIER, 'a'>
<T_DELIMITER, ';'>
<T_KEYWORD, 'float'>
<T_OPERATOR, '*'>
<T_IDENTIFIER, 'r'>
<T_OPERATOR, '='>
<T_OPERATOR, '&'>
<T_IDENTIFIER, 'e'>
<T_DELIMITER, ';'>
<T_KEYWORD, 'int'>
<T_IDENTIFIER, 's'>
<T_DELIMITER, ';'>
<T_KEYWORD, 'float'>
<T_IDENTIFIER, 't'>
<T_DELIMITER, ';'>
<T_KEYWORD, 'char'>
<T_IDENTIFIER, 'u'>
<T_DELIMITER, ';'>
<T_KEYWORD, 'int'>
<T_OPERATOR, '*'>
<T_IDENTIFIER, 'w'>
<T_DELIMITER, ';'>
<T_KEYWORD, 'return'>
<T_NUMBER, '0'>
<T_DELIMITER, ';'>
<T_DELIMITER, '}'>
